Projects/Addons/Sources/Views/ChangeAddonSummaryScreen.swift

```diff
@@ -49,9 +50,6 @@ extension ChangeAddonViewModel {
             ),
             isAddon: true
         ) {
-            Task {
```

Comment:
Removed this from here it was placed wrongly. It was calling submitAddons when we press "Add  and confirm" instead on bottom sheet on click on "Yes, upgrade my coverage".